README.md

```diff
@@ -53,7 +54,7 @@ These steps are provided for development purposes only.
    bin/console doctrine:schema:create
    ```
 7. Run a CRON job `bin/console packagist:run-workers` to make sure packages update.
-8. Run `npm run build` or `npm run dev` to build (or build&watch) css/js files.
+8. Run `npm run build` or `npm run dev` to build (or build&watch) css/js files. When using Docker run `docker-compose run node npm run dev` to watch css/js files.
```

Comment:
Shouldn't this be a space here? I know nothing but seems inconsistent :D

```suggestion
8. Run `npm run build` or `npm run dev` to build (or build&watch) css/js files. When using Docker run `docker compose run node npm run dev` to watch css/js files.
```

Reply:
oh yeah totally. Bad finger habit!

Reply:
docker-compose is legacy v1 where as docker compose (compose as a plugin to docker) is correct